plugins/module_utils/redfish_utils.py

```diff
@@ -2162,11 +2162,15 @@ def get_virtualmedia(self, resource_uri):
         result["entries"] = virtualmedia_results
         return result
 
-    def get_multi_virtualmedia(self):
+    def get_multi_virtualmedia(self, resource_type):
```

Comment:
For backwards compatibility, there should be a default value:
```suggestion
    def get_multi_virtualmedia(self, resource_type='Manager'):
```